app/assets/stylesheets/content/_index.scss

```diff
@@ -18,11 +18,13 @@
   }
 
   .table-header__param {
-    @include govuk-font($size: 19, $weight: bold);
+    @include govuk-font($size: 16, $weight: bold);
   }
 
   .govuk-table {
     background-color: $white;
+    // re-align text content of left and right columns
+    //margin-top: -1px;
```

Comment:
Maybe remove this?

Reply:
@cbaines Commenting it out was accidental - it's needed to make the two columns look right side by side (I was checking the effect in different browsers and accidentally left it commented out when I pushed.) Happy for it to go in still?

Reply:
Yeah, sure :+1: